ext/mbstring/tests/bug72399.phpt

```diff
@@ -8,8 +8,17 @@ if (!function_exists('mb_ereg')) die('skip mbregex support not available');
 --FILE--
 <?php
 $var5 = mb_ereg_search_init("","2");
+var_dump($var5);
 $var6 = mb_eregi_replace("2","","");
-$var13 = mb_ereg_search_pos();
+var_dump($var6);
+
+try {
+    $var13 = mb_ereg_search_pos();
+} catch (\Error $e) {
+    echo $e->getMessage() . \PHP_EOL;
+}
 ?>
---EXPECTF--
-Warning: mb_ereg_search_pos(): No regex given in %sbug72399.php on line %d
+--EXPECT--
+bool(true)
+string(0) ""
+No pattern was provided
```

Comment:
I really don't get why this throws, will need to investigate I suppose.

Reply:
Why shouldn't it throw?

Reply:
I don't know was confused, probably because I suppose it should throw before? As in in ``mb_ereg_search_init()`` ?